share/snis/commodities.txt

```diff
@@ -16,13 +16,16 @@
 gold ore, tons, metallic ore, 1010.0, 0.2, 1.0, 0.3, 0.2, 0.2, 10
 platinum ore, tons, metallic ore, 1480.0, 0.2, 1.0, 0.3, 0.2, 0.2, 5
 germanium ore, tons, semi-metallic ore, 490.0, 0.2, 1.0, 0.3, 0.2, 0.2, 30
-uranium ore, tons, uranium ore, 420.0, 0.2, 1.0, 0.3, 0.2, 0.2, 20
+uranium ore, tons, radioactive ore, 420.0, 0.2, 1.0, 0.3, 0.2, 0.2, 20
 gold, grams, gold, 10.0, 0.2, 1.0, 0.3, 0.2, 0.2, 10
 platinum, grams, platinum, 10.0, 0.2, 1.0, 0.3, 0.2, 0.2, 5
 germanium, kilograms, semi-metallic ore, 10.0, 0.2, 1.0, 0.3, 0.2, 0.2, 30
 uranium, grams, uranium, 10.0, 0.2, 1.0, 0.3, 0.2, 0.2, 20
 
 # "normal" items
+# name,    unit,    scans_as,    base_price, volatil., legality, econ_sens., govt_sens., tech_sens., odds
+refined sugar, kilograms, granulated organic crystals, 4.0, 0.2, 2.0, 0.2, 0.0, 0.0, 100
```

Comment:
The scans_as field is only 20 chars long.  "granulated organic crystals" is longer than 20 chars.